bestbook/static/build/css/style.css

```diff
@@ -85,9 +85,7 @@ select:focus, select:active, input[type=text]:focus, input[type=text]:active {
 }
 
 .block {
-    border: 5px solid #ccd6dd;
-    border-radius: 15px;
-
+    border-bottom: 1px dashed #233045;
     width: 100%;
     margin: 0 auto;
 }
```

Comment:
This class is used in several templates.  It may be better to revert these changes, create a new class with the `border-bottom` rule, and replace the `block` class in `browse.html` with the new class.